src/components/Articles/__tests__/createArticleForm.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Enzyme, { shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import CreateArticleFormContainer from '../CreateArticleFormContainer';
+import CreateArticleForm from '../CreateArticleFormComponent';
+
+Enzyme.configure({ adapter: new Adapter() });
+const middleWares = [thunk];
+const mockStore = configureStore(middleWares);
+let mountWrapper;
+let mountWrapperContainer;
+
+global.document = {
+  documentElement: {
+    classList: { add: jest.fn() },
+  },
+  getElementByClassname: jest.fn({id:'mde'}),
+};
+
+describe('Article creation form component/>', () => {
+  it('should match the snapshot new article creation form', () => {
+    const wrapper = shallow(<CreateArticleForm />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
The snapshot test is not necessary here. Also, one test should only test one thing. So, there should only be one `expect`.